src/save-data.js

```diff
@@ -1,4 +1,4 @@
-export var saveData = {
+export const saveData = {
```

Comment:
The idea was to have the default/zeroed out data so rather than having to write `this.characterData.thing1 = 0; this.characterData.thing2 = 0` etc. you can just say curData = saveData and it resets it

Reply:
Do you think I should rename it to "defaultData"?

Reply:
Yea defaultData would be much better

Reply:
Done